lms/views/lti/basic_launch.py

```diff
@@ -222,11 +222,19 @@ def _configure_js_to_show_document(self, document_url, assignment):
             if focused_user := self.request.params.get("focused_user"):
                 self.context.js_config.set_focused_user(focused_user)
 
-        elif assignment.is_gradable and self.request.lti_user.is_instructor:
-            # Only show the grading interface to teachers who aren't in Canvas,
-            # as Canvas uses its own built in Speedgrader
+        elif self.request.lti_user.is_instructor:
```

Comment:
Once/if we enable this for all non-canvas LMS I reckon it will be easier to move the decision to the product object instead of a setting.

While we develop/test/roll out it makes sense to use a setting :+1:

Reply:
> Once/if we enable this for all non-canvas LMS I reckon it will be easier to move the decision to the product object instead of a setting.

Do you mean moving the `hypothesis.edit_assignments_enabled` setting to a static property of the product which indicates whether the LMS supports assignment editing?